### PR TITLE
1: Change AzureRM.Profile to AzureRM.Accounts

### PR DESCRIPTION
**Pull request by MiYanni**
Updated usages of AzureRM.Profile to AzureRM.Accounts. The rename of AzureRM -> Az will be handled in the future across the entire repo. For now, we need `Profile` to say `Accounts` for tests to run in our module repo.

### CODE REVIEW

**Comment by MiYanni** [comment]
sign: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-common-sign/231/

File: src/Common/Properties/Resources.resx [hunk 1]
```diff
@@ -1473,8 +1473,8 @@ The file needs to be a PowerShell script (.ps1 or .psm1).</value>
 The data is anonymous and does not include commandline argument values.
 The data is collected by Microsoft.
 
-Use the Disable-AzureRmDataCollection cmdlet to turn the feature Off. The cmdlet can be found in the AzureRM.Profile module.  To disable data collection: PS &gt; Disable-AzureRmDataCollection.
-Use the Enable-AzureRmDataCollection cmdlet to turn the feature On. The cmdlet can be found in the AzureRM.Profile module.  To enable data collection: PS &gt; Enable-AzureRmDataCollection.</value>
+Use the Disable-AzureRmDataCollection cmdlet to turn the feature Off. The cmdlet can be found in the AzureRM.Accounts module.  To disable data collection: PS &gt; Disable-AzureRmDataCollection.
```
Comment (markcowl) [inline src/Common/Properties/Resources.resx]: Disable-AzDataCollection, Az.Accounts

File: src/ResourceManager/Properties/Resources.resx [hunk 2]
```diff
@@ -140,8 +140,8 @@
 The data is anonymous and does not include commandline argument values.
 The data is collected by Microsoft.
 
-Use the Disable-AzureRmDataCollection cmdlet to turn the feature Off. The cmdlet can be found in the AzureRM.Profile module.  To disable data collection: PS &gt; Disable-AzureRmDataCollection.
-Use the Enable-AzureRmDataCollection cmdlet to turn the feature On. The cmdlet can be found in the AzureRM.Profile module.  To enable data collection: PS &gt; Enable-AzureRmDataCollection.</value>
+Use the Disable-AzureRmDataCollection cmdlet to turn the feature Off. The cmdlet can be found in the AzureRM.Accounts module.  To disable data collection: PS &gt; Disable-AzureRmDataCollection.
```
Comment (markcowl) [inline src/ResourceManager/Properties/Resources.resx]: same comments

File: src/Common/Properties/Resources.resx (hunk 1 above)
Comment (MiYanni) [reply src/Common/Properties/Resources.resx]: I *specifically stated* in the PR description that AzureRM -> Az would be done at a later date (to preserve consistency). But, I still updated this anyway to address the comment.

File: src/ResourceManager/Properties/Resources.resx (hunk 2 above)
Comment (MiYanni) [reply src/ResourceManager/Properties/Resources.resx]: Fixed.

File: src/Common/Properties/Resources.Designer.cs [hunk 3]
```diff
@@ -236,8 +236,8 @@ public class Resources {
         ///The data is anonymous and does not include commandline argument values.
         ///The data is collected by Microsoft.
         ///
-        ///Use the Disable-AzureRmDataCollection cmdlet to turn the feature Off. The cmdlet can be found in the AzureRM.Profile module.  To disable data collection: PS &gt; Disable-AzureRmDataCollection.
-        ///Use the Enable-AzureRmDataCollection cmdlet to turn the feature On. The cmdlet can be found in the AzureRM.Profile module.  To enable  [rest of string was truncated]&quot;;.
+        ///Use the Disable-AzureRmDataCollection cmdlet to turn the feature Off. The cmdlet can be found in the AzureRM.Accounts module.  To disable data collection: PS &gt; Disable-AzureRmDataCollection.
```
Comment (markcowl) [inline src/Common/Properties/Resources.Designer.cs]: Same comment

File: src/ResourceManager/Properties/Resources.Designer.cs [hunk 4]
```diff
@@ -65,8 +65,8 @@ public class Resources {
         ///The data is anonymous and does not include commandline argument values.
         ///The data is collected by Microsoft.
         ///
-        ///Use the Disable-AzureRmDataCollection cmdlet to turn the feature Off. The cmdlet can be found in the AzureRM.Profile module.  To disable data collection: PS &gt; Disable-AzureRmDataCollection.
-        ///Use the Enable-AzureRmDataCollection cmdlet to turn the feature On. The cmdlet can be found in the AzureRM.Profile module.  To enable  [rest of string was truncated]&quot;;.
+        ///Use the Disable-AzureRmDataCollection cmdlet to turn the feature Off. The cmdlet can be found in the AzureRM.Accounts module.  To disable data collection: PS &gt; Disable-AzureRmDataCollection.
```
Comment (markcowl) [inline src/ResourceManager/Properties/Resources.Designer.cs]: Same comment - I thought these would be the same as below

File: src/Common/Properties/Resources.Designer.cs (hunk 3 above)
Comment (MiYanni) [reply src/Common/Properties/Resources.Designer.cs]: Fixed.

File: src/ResourceManager/Properties/Resources.Designer.cs (hunk 4 above)
Comment (MiYanni) [reply src/ResourceManager/Properties/Resources.Designer.cs]: Fixed.

**Pull request merged**